src/main/java/com/feedzai/commons/sql/abstraction/ddl/DbColumnType.java

```diff
@@ -51,6 +51,11 @@
      */
     CLOB,
     /**
+     * The json type
```

Comment:
I think it's important to note that this is not natively supported by all engines and is stored as a CLOB as last resort.